db/migrate/20200930080214_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
```

Comment:
Add indexes and constraints, please

Reply:
@sviatoslav-krupa fixed index by email, is it ok?

Reply:
okay, but what about constraints?